scipy/stats/tests/test_survival.py

```diff
@@ -337,3 +337,56 @@ def test_right_censored_against_uncensored(self):
         assert_equal(res.sf._n, ref.sf._n)
         assert_equal(res.sf._d[:-1], ref.sf._d[:-1])  # difference @ [-1]
         assert_allclose(res.sf._sf[:-1], ref.sf._sf[:-1], rtol=1e-14)
+
+
+class TestLogRank:
+
+    @pytest.mark.parametrize(
+        "x, y, statistic",
+        # uncensored, censored
+        [(
+              # https://sphweb.bumc.bu.edu/otlt/mph-modules/bs/bs704_survival/BS704_Survival5.html  # noqa
+              [[8, 12, 26, 14, 21, 27], [8, 32, 20, 40]],
+              [[33, 28, 41], [48, 48, 25, 37, 48, 25, 43]],
+              6.148087536256203
```

Comment:
The reference shows 6.151, but this is precisely the number that the code produces. I hope this is not using the output of the code as a reference : /

6.151 may be inaccurate due to roundoff error. In this case, the thing to do is to compute the value with an external implementation and include that code as a comment.